ftw/permissionmanager/testing.py

```diff
@@ -31,10 +32,13 @@ def setUpZope(self, app, configurationContext):
             context=configurationContext)
 
         z2.installProduct(app, 'ftw.permissionmanager')
+        z2.installProduct(app, 'Products.DateRecurringIndex')
 
     def setUpPloneSite(self, portal):
         # Install into Plone site using portal_setup
         applyProfile(portal, 'ftw.permissionmanager:default')
+        applyProfile(portal, 'plone.app.contenttypes:default')
```

Comment:
We usually only install `p.a.contenttypes` with Plone 5, not with Plone 4.
Is it intentional that we are installing it here for Plone 4 too?

Reply:
Since I've dropped archetypes for Plone 4 as well, I don't want to test my functions with archetypes for Plone 4. This package replaces `Products.ATContentTypes`, so I don't understand why we shouldn't need it for Plone 4